device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ModuleClient.java

```diff
@@ -368,6 +368,7 @@ public static ModuleClient createFromEnvironment(IotHubClientProtocol protocol,
         }
     }
 
+    @SuppressWarnings("SameParameterValue") // The SEND_PERIOD is currently 10ms for all protocols, but can be made configurable in the future.
```

Comment:
This is private so we always have option to change signature if the value is hardcoded. Let me know your thoughts.

Reply:
yes, we can, but if in the future we decide to make the send period configurable, we'll need to change the signature again.

Reply:
There were other places where the value is hardcoded.